blurp-app/src/components/google_signin.jsx

```diff
@@ -46,12 +49,36 @@ function GoogleLoginButton (props) {
   // If the profile changes (logged in, logged out, image change, etc),
   // then we should change what is rendered
   useEffect(() => {
-    if(profile && profile != 'null')
-      setRenderedContent(userProfile(false));
-    else
+    if(profile && profile != 'null') {
+      setPopoutVisible(false);
+      setRenderedContent(userProfile());
+    }
+    else {
       setRenderedContent(signInButton);
+    }
   }, [profile]);
 
+  // Collapse popout if user clicks outside
+  useEffect(() => {
+    const handleClickOutside = (event) => {
+      if(expanded_div_ref.current && !expanded_div_ref.current.contains(event.target) &&
+        profile_pic_ref.current && !profile_pic_ref.current.contains(event.target)) {
+        closePopout && closePopout();
```

Comment:
I know that you mentioned that the changes here were intended for another commit, but wanted to point out that this line looks weird, you probably only need closePopout()

Reply:
Oh yeah, closePopout is guaranteed to be defined since it's in this component. I'll remove that